Examples/restcomm-olympus-nopod/restcomm-olympus/Utils.m

```diff
@@ -34,20 +51,20 @@ + (void) setupUserDefaults
     
     // DOC: very important. To add a NSDictionary or NSArray as part of NSUserDefaults the key must always be an NSString!
     NSDictionary *basicDefaults = @{
-                                    @"is-first-time" : @(YES),
-                                    @"pending-interapp-uri" : @"",  // has another app sent us a URL to call?
-                                    @"sip-identification" : @"",  //@"sip:ios-sdk@cloud.restcomm.com",
-                                    @"sip-password" : @"",
-                                    @"sip-registrar" : @"cloud.restcomm.com",
-                                    @"turn-enabled" : @(YES),
-                                    @"turn-url" : @"https://service.xirsys.com/ice",  // @"https://computeengineondemand.appspot.com/turn",
-                                    @"turn-username" : @"atsakiridis",  // @"iapprtc",
-                                    @"turn-password" : @"4e89a09e-bf6f-11e5-a15c-69ffdcc2b8a7",  // @"4080218913"
-                                    @"signaling-secure" : @(YES),  // by default signaling is secure
-                                    @"signaling-certificate-dir" : @"",
+                                    kIsFirstTimeKey : @(YES),
+                                    kPendingInterappKey : @"",  // has another app sent us a URL to call?
+                                    kSipIndentificationKey : @"",  //@"sip:ios-sdk@cloud.restcomm.com",
+                                    kSipPasswordKey : @"",
+                                    kSipRegistrarKey : @"cloud.restcomm.com",
+                                    kTurnEnabledKey : @(YES),
+                                    kTurnUrlKey : @"https://service.xirsys.com/ice",  // @"https://computeengineondemand.appspot.com/turn",
```

Comment:
Lets use a xirsys V3 url as discussed in my other comment for kTurnUrlKey. I believe it should be:

`https://es.xirsys.com/_turn/`

Just make sure that you test it ;)